### ISSUE TITLE
"getting started icon"

### ISSUE THREAD

**Issue by d-v-b** [zarr-developers/zarr-python#2569]
I'm not a fan of the icon over the "getting started" section of the docs landing page, it conveys "running away from danger" to me much more than "getting started with something":

![Image](https://github.com/user-attachments/assets/35266e58-4fd1-4b60-ab54-a931811f74ed)

 IMO the text "getting started" alone is sufficient, and so we could just remove that icon entirely. Any thoughts?

**Comment by jhamman** [comment]
I don't think this is a big deal. You'll find that same icon in a number of places around the ecosystem:
 
- https://docs.xarray.dev/en/stable/index.html
- https://pandas.pydata.org/docs/

Certainly open to replacing the icon but doesn't seem like much of a priority to me.

**Comment by d-v-b** [comment]
I find that icon is weird in all those other places too. But I do think my bigger objection is to the layout itself -- "getting started" is actually the job of the main page and so I don't think it needs its own category. Also, I don't think the other categories should be displayed in a grid, each with their own large icons, etc. I agree that it's not a big priority right now, but I would very much support restructuring our docs to be organized more like pydantic's

**Comment by jhamman** [comment]
Just a quick note to mention that all of these sites are attempting to follow the Diátaxis framework: https://diataxis.fr/

Not saying we're doing it super well but that is the origin for this layout.

**Comment by DimitriPapadopoulos** [comment]
The current icons don't help, they're a distraction:
- The runner is used in [ISO emergency exit](https://www.iso.org/obp/ui#iso:grs:7010:E002) pictograms. It feels more like "The End" than "Getting Started" to me!
- The book icon 🕮 could apply to the Contributor's Guide as well as to the Tutorial.
- I find `[...]` does not relate to "API Reference".
- It took me quite some time to understand that `>_` is a prompt, hence its use as an icon for the Contributor's Guide — but contributors don't necessarily use a prompt.

Instead, I'd rather see some nice coloured background for each type of documentation, or just nice unrelated images — like O'Reilly books. Or just no icons/images.

The black boxes, with the _To the getting started guide_ or _To the api reference guide_ links, are redundant and a distraction as well. The whole surrounding box is a link itself! Why add a link inside a link?

**Comment by DimitriPapadopoulos** [comment]
It would be great if a "dev" documentation could be made available under https://zarr.readthedocs.io, but not necessarily as the default.

**Comment by d-v-b** [comment]
that icon is gone after our docs overhaul, so this can be closed
